### DISCUSSION TITLE
Multiple cursor repo is deprecated

### DISCUSSION THREAD

**Discussion by ahmadshiddiq** [JetBrains/ideavim#642] [General]
I read the repo from terryma for multiple cursor is deprecated and not updated since 2020, and even terryma repo said we should using another one that still up to date.
Is there any plan to update it?

https://github.com/mg979/vim-visual-multi

**Comment by AlexPl292** [comment]
We have plans, but it's not implemented yet: [VIM-2293](https://youtrack.jetbrains.com/issue/VIM-2293)
